test/signs/realtime_test.exs

```diff
@@ -162,6 +162,10 @@ defmodule Signs.RealtimeTest do
         datetime(~T[05:00:00])
       end)
 
+      stub(Engine.ScheduledHeadways.Mock, :get_last_scheduled_departure, fn _ ->
+        datetime(~D[2023-01-02], ~T[02:00:00])
```

Comment:
@PaulJKim just wanted to clarify here - we'll never have a case where the first scheduled departure is after the last scheduled departure right? So making this explicit in the tests is okay? 

This comes into play with the `before_early_am_threshold` with the after last scheduled departure logic within `overnight_period?` and wanted to make sure we never get into the weird state where this might happen.